File: csharp/dsf/dsf/src/DSFrameworkException.cs
using System;

namespace com.baidu.dsf
{

	/// 
	/// <summary>
	/// @author arksea
	/// </summary>
	public class DSFrameworkException : Exception
	{

		public DSFrameworkException(string msg) : base(msg)
		{
		}

		public DSFrameworkException(string msg, Exception ex) : base(msg, ex)
		{
		}
	}

}

File: java/dsframework/src/main/java/dsf/DSFrameworkException.java
package dsf;

/**
 *
 * @author arksea
 */
public class DSFrameworkException extends Exception {

    public DSFrameworkException(String msg) {
        super(msg);
    }

    public DSFrameworkException(String msg, Throwable ex) {
        super(msg, ex);
    }
}
